src/Navigation.js

```diff
@@ -1,131 +1,49 @@
-                        <>
-                            <Nav.Item>
-                                <Card>
-                                    <Card.Header>
-                                        <Accordion.Toggle as={Link} to="/register" eventKey="0">Register</Accordion.Toggle>
-                                    </Card.Header>
-                                </Card>
-                            </Nav.Item>
-                            <Nav.Item>
-                                <Card>
-                                    <Card.Header>
-                                        <Accordion.Toggle as={Link} to="/login" eventKey="0" onClick={login}>Login</Accordion.Toggle>
-                                    </Card.Header>
-                                </Card>
-                            </Nav.Item>
-                        </>
-                        :
-                        <Nav.Item>
-                            <Card>
-                                <Card.Header>
-                                    <Accordion.Toggle as={Link} to="/" eventKey="0" onClick={logout}>Logout</Accordion.Toggle>
-                                </Card.Header>
-                            </Card>
-                        </Nav.Item>
-                    }
-                    {isAuth && <Nav.Item>
-                        <Card>
-                            <Card.Header>
-                                <Accordion.Toggle as={Link} to="/" eventKey="0">Welcome back, {user}!</Accordion.Toggle>
-                            </Card.Header>
-                        </Card>
-                    </Nav.Item>}
+      <div className="topnav">
+               <Navbar className="ml-auto" expand="lg" style={{borderTop:' 2px solid #54BCEB'}}>
+               <Navbar.Toggle aria-controls='responsive-navbar-nav' />
+               <Navbar.Collapse id="responsive-navbar-nav">
+               <Nav className="mr-auto">
+                 <div className="nav-items">
+                      <Link to="/" className="li">
+                         <li >About Us</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li >Services</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li >Membership</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li>FAQs</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li>Blogs</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li>Contact Us</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li className="li_signIn">Sign In</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li className="active" >Sign Up</li>
```

Comment:
hello, I guess you have applied to the Active class only to Sign UP. I believe all the list elements in nav will have that active class on hover.
Apart from that things look good to me.
![BITtesting](https://user-images.githubusercontent.com/58786336/116027578-e39afd00-a672-11eb-9e00-3294f10f6e69.gif)
Am I right @Rahulm2310 @mtreacy002 ?

Reply:
@shades-7 not sure about it. @VectoTec @Vuyanzi can you please tell what's the expected behaviour.

Reply:
@shades-7 According to the Figma file, a blue line appears upon hover. Active class is for the blue background of the signup button.

Reply:
Here you can link to the existing `Register` page until we have the new `Sign Up` page created so not to block the contributor to access the `Register` feature.